scripts/stack.ts

```diff
@@ -98,7 +98,13 @@ class MyStack extends Stack {
         visibilityTimeout: Duration.minutes(3),
       })
       const cfnQueue = newQueue.node.defaultChild as CfnResource
-      cfnQueue.overrideLogicalId(ref)
+      if (ref == "ResultQueue") {
+        cfnQueue.overrideLogicalId("ResultQueue98CD34E0")
+      } else if (ref == "ErrorQueue") {
+        cfnQueue.overrideLogicalId("ErrorQueue2580A2D4")
+      } else {
+        cfnQueue.overrideLogicalId(ref)
+      }
```

Comment:
Would these conditions only be hit on the initial deploy? If so, I wonder if we could utilize a koala command instead.

Reply:
This will get hit on every environment deployment as the cloudformation stack needs to be created per environment.

Reply:
Yes, that makes sense. To clarify, I mean after each initial environment deploy.

Reply:
While it could work that way, I think it would be better practice to have the stack be rendered on each deployment in case that there are any changes to the IAC. This also ensure idempotency of the infrastructure that gets deployed.